### PR TITLE
branch_one

### PR DESCRIPTION
**Pull request by projectImplementer**
Test

### CODE REVIEW

File: src/test/java/softvisionProject/POM/CareersPage.java [hunk 1]
```diff
@@ -37,28 +37,48 @@ public CareersPage(WebDriver driver) {
...
-            }
-            }
+                for (String location: dropDownList.keySet()) {
+                    if (location.contains(name)) {
+                    System.out.println("location by name found");
+                    System.out.println("Name taken " + name);
+                    dropDownList.get(location).click();
+                    break;
+                    } else {
+                    System.out.println("Nothing to click on");
```
Comment (cjuncanariu) [inline src/test/java/softvisionProject/POM/CareersPage.java]: change the error message with a specific message for the user

File: src/test/java/softvisionProject/POM/CareersPage.java [hunk 2]
```diff
@@ -37,28 +37,48 @@ public CareersPage(WebDriver driver) {
...
+                }
         } else {
                 System.out.println("- location display: fail");
         }
     }
+
+    // HashMap methods - get the EXPERTISE / GUILDS drop down list items
+    public void selectExpertise(String name) {
+        if (expertisez.isDisplayed()) {
+            System.out.println("- expertise display: pass");
```
Comment (cjuncanariu) [inline src/test/java/softvisionProject/POM/CareersPage.java]: expertize is not pass, is displayed.

File: src/test/java/softvisionProject/POM/CareersPage.java [hunk 3]
```diff
@@ -37,28 +37,48 @@ public CareersPage(WebDriver driver) {
...
+            System.out.println("- expertise display clicked");
+            JavaMap test = new JavaMap();
+            HashMap <String, WebElement> expertiseList = test.getExpertiseList(driver);
+                for(String expertise: expertiseList.keySet()) {
+                    if (expertise.contains(name)) {
+                        System.out.println("Expertise found : " + name);
+                        expertiseList.get(expertise).click();
+                        break;
+                    } else {
+                    System.out.println("Nothing to click on");
```
Comment (cjuncanariu) [inline src/test/java/softvisionProject/POM/CareersPage.java]: expecting a specific message for the user. "Nothing to click on" do not tell me too much

File: src/test/java/softvisionProject/POM/CareersPage.java [hunk 4]
```diff
@@ -82,9 +102,9 @@ public void clickLocation(String selectLocation) {
     }
 
     public void clickExpertise(String selectExpertise) {
```
Comment (cjuncanariu) [inline src/test/java/softvisionProject/POM/CareersPage.java]: What is the diff between clickExpertise and selectExpertise ?

**Review by cjuncanariu — CHANGES_REQUESTED** [review]

File: src/test/java/softvisionProject/POM/CareersPage.java (hunk 4 above)
Comment (projectImplementer) [reply src/test/java/softvisionProject/POM/CareersPage.java]: clickExpertise is the old method for dealing with drop down lists, similar to clickLocations which is now being replaced by selectLocations

Comment (cjuncanariu) [reply src/test/java/softvisionProject/POM/CareersPage.java]: if is old remove it

File: src/test/java/softvisionProject/POM/CareersPage.java (hunk 2 above)
Comment (projectImplementer) [reply src/test/java/softvisionProject/POM/CareersPage.java]: Changed. Will update in a few minutes.

File: src/test/java/softvisionProject/POM/CareersPage.java (hunk 3 above)
Comment (projectImplementer) [reply src/test/java/softvisionProject/POM/CareersPage.java]: Changed. Will update shortly.

**Pull request merged**